File: tmp_make_logo_transparent.php
<?php
$src = __DIR__ . DIRECTORY_SEPARATOR . 'public' . DIRECTORY_SEPARATOR . 'sda logo.png';
$out = __DIR__ . DIRECTORY_SEPARATOR . 'public' . DIRECTORY_SEPARATOR . 'sda-logo-transparent.png';

if (!extension_loaded('gd')) {
    fwrite(STDERR, "GD extension is not enabled.\n");
    exit(1);
}

$image = @imagecreatefrompng($src);
if (!$image) {
    fwrite(STDERR, "Failed to open source image: {$src}\n");
    exit(1);
}

$width = imagesx($image);
$height = imagesy($image);

imagealphablending($image, false);
imagesavealpha($image, true);

for ($y = 0; $y < $height; $y++) {
    for ($x = 0; $x < $width; $x++) {
        $index = imagecolorat($image, $x, $y);
        $rgba = imagecolorsforindex($image, $index);

        $r = $rgba['red'];
        $g = $rgba['green'];
        $b = $rgba['blue'];

        // Make near-white pixels fully transparent
        if ($r > 240 && $g > 240 && $b > 240) {
            $transparent = imagecolorallocatealpha($image, $r, $g, $b, 127);
            imagesetpixel($image, $x, $y, $transparent);
        }
    }
}

if (!imagepng($image, $out)) {
    imagedestroy($image);
    fwrite(STDERR, "Failed to save output image: {$out}\n");
    exit(1);
}

imagedestroy($image);

echo $out . PHP_EOL;

File: tmp_make_logo_transparent.py
from PIL import Image

src = r"c:\Users\STUDENTS\Desktop\sda\public\sda logo.png"
out = r"c:\Users\STUDENTS\Desktop\sda\public\sda-logo-transparent.png"

img = Image.open(src).convert("RGBA")
pixels = img.getdata()
new_pixels = []

for r, g, b, a in pixels:
    # Remove near-white background while preserving logo details
    if r > 240 and g > 240 and b > 240:
        new_pixels.append((r, g, b, 0))
    else:
        new_pixels.append((r, g, b, a))

img.putdata(new_pixels)
img.save(out, "PNG")
print(out)
